src/pages/components/Count/Count.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import './Count.scss';
+
+// const Count = {counter, setCount} => {}
+
+const Count = props => {
+  const { count, setCount } = props;
```

Comment:
```suggestion
const Count =({ count, setCount }) => {
```

함수형 컴포넌트 인자 자리에서 이와 같이 구조분해할당을 할 수 있습니다!